### PR TITLE
some refactoring

### PR DESCRIPTION
**Pull request by ukH9YFxczKfxEnNECPyq**
as i think [[ ]] && is  more efficient than if and one case

### CODE REVIEW

File: bbb-install.sh [hunk 1]
```diff
@@ -725,9 +719,8 @@ HERE
 
 
 install_ssl() {
-  if [ -f /var/www/bigbluebutton/client/conf/config.xml ]; then
-    sed -i 's/tryWebRTCFirst="false"/tryWebRTCFirst="true"/g' /var/www/bigbluebutton/client/conf/config.xml
-  fi
+    [ -f /var/www/bigbluebutton/client/conf/config.xml ] \
+	& sed -i 's/tryWebRTCFirst="false"/tryWebRTCFirst="true"/g' /var/www/bigbluebutton/client/conf/config.xml
```
Comment (ak4t0sh) [inline bbb-install.sh:723]:
```suggestion
	&& sed -i 's/tryWebRTCFirst="false"/tryWebRTCFirst="true"/g' /var/www/bigbluebutton/client/conf/config.xml
```

**Comment by ak4t0sh** [comment]
Thanks for sharing. I have added a code suggestion.
I agree that keeping code as short as possible is important but IMO in this case it makes code less readable.

**Comment by ukH9YFxczKfxEnNECPyq** [comment]
i believe that good comment helps write less code

**Comment by pyloor** [comment]
> i believe that good comment helps write less code
wat? other way works: good code needs less comments.

**Comment by ukH9YFxczKfxEnNECPyq** [comment]
well  maybe i am wrong but isn't
[[ true ]] \
  && command
is more like clean code than
if [[ true ]]; then
  command
fi
?

**Pull request closed without merging**